### PR TITLE
base update to 2.2

### PR DESCRIPTION
**Pull request by kelseiv**
Closes #

### CODE REVIEW

File: data/products.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ influxdb:
   namespace: influxdb
   list_order: 2
   versions: [v1.3, v1.4, v1.5, v1.6, v1.7, v1.8, v2.0, v2.1]
```
Comment (sanderson) [inline data/products.yml]:
```suggestion
  versions: [v1.3, v1.4, v1.5, v1.6, v1.7, v1.8, v2.0, v2.1, v2.2]
```

File: data/products.yml [hunk 2]
```diff
@@ -4,8 +4,8 @@ influxdb:
...
-  latest: v2.1
-  latest_override: v2.1
+  latest: v2.2
+  latest_override: v2.2
   latest_patches:
     "2.1": 1
```
Comment (sanderson) [inline data/products.yml]:
```suggestion
  latest_patches:
    "2.2": 0
    "2.1": 1
```

**Review by sanderson — APPROVED** [review]
From a really quick look-over, I only noticed two changes that need to be made.

**Comment by kelseiv** [comment]
cc @samhld

**Pull request merged**